apis/python/src/tiledbsc/soma.py

```diff
@@ -547,6 +547,98 @@ def from_soma_slice(
 
         return soma
 
+    @classmethod
+    def map(
+        cls,
+        somas: List[SOMA],
```

Comment:
does this need to be a `List` in particular, or will any `Iterable` (or, if you need to be slightly more specific, `Sequence`) work here?